crates/bevy_asset/src/processor/log.rs

```diff
@@ -32,8 +32,10 @@ pub struct ProcessorTransactionLog {
 /// An error that occurs when reading from the [`ProcessorTransactionLog`] fails.
 #[derive(Error, Debug)]
```

Comment:
Also might be worth linking to this above https://github.com/bevyengine/bevy/blob/main/crates/bevy_asset/src/processor/mod.rs